scripts/e2e_local.sh

```diff
@@ -28,4 +28,4 @@ export MCLI_OPS_MANAGER_URL="https://cloud-qa.mongodb.com/"
 export MCLI_PUBLIC_API_KEY="${public_key}"
 export MCLI_PRIVATE_API_KEY="${private_key}"
 export MCLI_ORG_ID="${org_id}"
-ginkgo --focus "${focus_key}" -nodes=3 -x -v test/e2e/
+ginkgo -focus "${focus_key}" -stream -nodes=6 -x -v test/e2e/
```

Comment:
[q] what does `stream` do?

Reply:
it will enumerate each node test and print it as is - node output will be mixed (for configuration test only). So I was thinking to leave it like this and see if there are any more failures